aws/policy/security-services.yaml

```diff
@@ -95,6 +95,20 @@ Statement:
       - kms:UpdateKeyDescription
     Resource: "*"
 
+  - Sid: AllowGlobalRestrictedResourceActionsWhichIncurFees
+    Effect: Allow
+    Action:
+      - secretsmanager:DescribeSecret
+      - secretsmanager:ListSecrets
```

Comment:
Both Describe and List need to be in the above group (`AllowGlobalUnrestrictedResourceActionsWhichIncurNoFees`, operating on `Resource: "*"`)